bootstrapper-maven-plugin/src/main/resources/templates/CustomResource.java

```diff
@@ -0,0 +1,11 @@
+package {{groupId}};
+
+import io.fabric8.kubernetes.client.CustomResource;
+import io.fabric8.kubernetes.api.model.Namespaced;
+import io.fabric8.kubernetes.model.annotation.Group;
+import io.fabric8.kubernetes.model.annotation.Version;
+
+@Group("{{groupId}}")
+@Version("v1")
```

Comment:
I think the version should also be configurable.

Reply:
Might, be but would like to do such additions, since not really part of mvp